src/components/contextual/pages/pool/PoolChart.vue

```diff
@@ -391,6 +402,28 @@ function setCurrentChartValue(payload: {
     PRETTY_DATE_FORMAT
   );
 }
+
+function addLackingTimestamps() {
```

Comment:
I think the word should be lagging right?
```suggestion
function addLaggingTimestamps() {
```

Reply:
yes, its better here, fixed